Lib/configparser.py

```diff
@@ -1006,7 +1009,7 @@ def _read(self, fp, fpname):
         Configuration files may include comments, prefixed by specific
         characters (`#' and `;' by default). Comments may appear on their own
         in an otherwise empty line or may be entered in lines holding values or
-        section names.
+        section names. Please note that comments get stripped off on reading.
```

Comment:
**Please note that comments get stripped off when reading configuration files.**